index.html

```diff
@@ -8,8 +8,24 @@
     </head>
     </body>
         <main>
+            <div id="root"></div>
         </main>
-        <script src="index.js"></script>
+            <script src="http://unpkg.com/react@16.7.0/umd/react.production.min.js"></script>
+            <script src="http://unpkg.com/react-dom@16.7.0/umd/react-dom.production.min.js"></script>
+            <script src="https://unpkg.com/@babel/standalone@7.8.3/babel.js"></script>
+            <script type="text/babel">
+                
+                class App extends React.Component {
```

Comment:
Awesome! Looks good. Just an aside, class components are slowly getting used less and less and being replaced by functional components widely. Something to keep in mind!